DatadogTrace/Sources/Integrations/TracingURLSessionHandler.swift

```diff
@@ -90,7 +90,7 @@ internal struct TracingURLSessionHandler: DatadogURLSessionHandler {
     func interceptionDidComplete(interception: DatadogInternal.URLSessionTaskInterception) {
         guard
             interception.isFirstPartyRequest,  // `Span` should be only send for 1st party requests
-            !interception.rum2APM, // Send a span only if RUM 2 APM is disabled
+            interception.origin != "rum", // Send a span only if origin is not 'rum'
```

Comment:
nit/ This comment repeats exactly what the code says 🙂. I believe a better context for the reader would be to explain the RUM <> Tracing game.
```suggestion
            interception.origin != "rum", // if that request was tracked as RUM resource, the RUM backend will create the span on our behalf
```